README.md

```diff
@@ -58,6 +58,7 @@ jobs:
       isPush: ${{ steps.workflowEnv.outputs.isPush }}
       isTagged: ${{ steps.workflowEnv.outputs.isTagged }}
       isRelease: ${{ steps.workflowEnv.outputs.isRelease }}
+      isNewest: ${{ steps.workflowEnv.outputs.isNewest }}
```

Comment:
I'd use `isLatest` terminology as it checks whether the CI is running the latest code

Reply:
so I thought about using `isLatest` but as per github, the tag with `latest` is the latest release and not pre-released release created after `latest` release. I can change it to `isLatest` I am already annoyed with it.

Reply:
Haha, my head exploded from so many occurrences of `latest` in one sentence. I think there is a misunderstanding of what we are trying to surface. This action can be run on any branch, not only on a tag. So the goal is to check that the code is the latest (aka is the same as in the default branch).

Reply:
well that make sense now :) updated the variable to `isLatest` everywhere.